src/_base/mutagen.yml.twig

```diff
@@ -1,10 +1,10 @@
 forward:
-  teufel:
+  {{ @('workspace.name') }}:
```

Comment:
🤦‍♂ , thanks for fixing this, I totally missed it

Reply:
NP :)